### DISCUSSION TITLE
Can i join team?

### DISCUSSION THREAD

**Discussion by tedomi2705** [ROM-builders/temporary#18342] [General]
Here my trees

Device Tree
https://github.com/Evolution-X-Devices/device_xiaomi_tulip

Kernel Tree
https://github.com/Evolution-X-Devices/kernel_xiaomi_sdm660

Vendor Tree
https://github.com/Evolution-X-Devices/vendor_xiaomi_tulip

My successful build
https://cirrus-ci.com/build/4829028243210240
https://cirrus-ci.com/build/5053392346152960

Telegram username: https://t.me/tedomi2705

**Comment by Apon77** [comment]
seems like your account is inactive.

**Reply by tedomi2705** [reply]
I build for evo only and currently evo source is still merging qpr2

**Reply by tedomi2705** [reply]
Am i able to join now?
